Error while checking build/O2/fullCI_slc9 for f4c88b9beb6dfcb541bbc4ed1b8a779f26906ba3 at 2026-04-14 20:37:
```
## sw/BUILD/O2-latest/log
/sw/SOURCES/O2/slc9_x86-64-slc9_x86-64/0/GPU/GPUTracking/DataTypes/TPCPadGainCalib.cxx:50:29: error: 'TPC_PADS_IN_SECTOR' was not declared in this scope; did you mean 'TPC_REAL_PADS_IN_SECTOR'?
/sw/SOURCES/O2/slc9_x86-64-slc9_x86-64/0/GPU/GPUTracking/DataTypes/TPCPadBitMap.cxx:43:29: error: 'TPC_PADS_IN_SECTOR' was not declared in this scope; did you mean 'TPC_REAL_PADS_IN_SECTOR'?
/sw/SOURCES/O2/slc9_x86-64-slc9_x86-64/0/GPU/GPUTracking/DataTypes/CalibdEdxContainer.cxx:268:42: error: 'TPC_PADS_IN_SECTOR' was not declared in this scope; did you mean 'TPC_FEC_IDS_IN_SECTOR'?
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/15278/f4c88b9beb6dfcb541bbc4ed1b8a779f26906ba3/build_O2_fullCI_slc9/pretty.html).

--- a/GPU/GPUTracking/DataTypes/TPCPadBitMap.cxx
+++ b/GPU/GPUTracking/DataTypes/TPCPadBitMap.cxx
@@ -40,7 +40,7 @@ TPCPadBitMap::TPCPadBitMap(const o2::tpc::CalDet<bool>& map) : TPCPadBitMap()
 void TPCPadBitMap::setFromMap(const o2::tpc::CalDet<bool>& map)
 {
   for (int32_t sector = 0; sector < o2::tpc::constants::MAXSECTOR; sector++) {
-    for (int32_t p = 0; p < TPC_PADS_IN_SECTOR; p++) {
+    for (int32_t p = 0; p < TPC_REAL_PADS_IN_SECTOR; p++) {
       const auto val = map.getValue(sector, p);
       mBitMap[sector].set(p, val);
     }

--- a/GPU/GPUTracking/DataTypes/TPCPadGainCalib.cxx
+++ b/GPU/GPUTracking/DataTypes/TPCPadGainCalib.cxx
@@ -47,7 +47,7 @@ TPCPadGainCalib::TPCPadGainCalib(const o2::tpc::CalDet<float>& gainMap, const fl
 void TPCPadGainCalib::setFromMap(const o2::tpc::CalDet<float>& gainMap, const bool inv)
 {
   for (int32_t sector = 0; sector < o2::tpc::constants::MAXSECTOR; sector++) {
-    for (int32_t p = 0; p < TPC_PADS_IN_SECTOR; p++) {
+    for (int32_t p = 0; p < TPC_REAL_PADS_IN_SECTOR; p++) {
       const float gainVal = gainMap.getValue(sector, p);
       inv ? mGainCorrection[sector].set(p, (gainVal > 1.e-5f) ? 1.f / gainVal : 1.f) : mGainCorrection[sector].set(p, gainVal);
     }

--- a/GPU/GPUTracking/DataTypes/TPCZSLinkMapping.cxx
+++ b/GPU/GPUTracking/DataTypes/TPCZSLinkMapping.cxx
@@ -27,7 +27,7 @@ TPCZSLinkMapping::TPCZSLinkMapping(o2::tpc::Mapper& mapper)
   assert(fecToGlobalPad.size() == TPC_FEC_IDS_IN_SECTOR);
 
   const auto& globalPadToPadPos = mapper.getMapGlobalPadToPadPos();
-  assert(globalPadToPadPos.size() == TPC_PADS_IN_SECTOR);
+  assert(globalPadToPadPos.size() == TPC_REAL_PADS_IN_SECTOR);
 
   for (size_t i = 0; i < TPC_FEC_IDS_IN_SECTOR; i++) {
     FECIDToPadPos[i] = globalPadToPadPos[fecToGlobalPad[i]];

